app/scripts/pages/project.jsx

```diff
@@ -13,6 +13,7 @@ import { Path } from '../util/urlEnum';
 class Project extends React.Component {
 
     componentDidMount() {
+        mainStore.setListItems([]);
```

Comment:
What is this doing? Do we need it? I fixed the "navigation from Dashboard to Projects" (which was actually an issue navigating into a project) by adding a check in listItems.jsx.

```let children = items && items.length && !items.some(i => i.kind === Kind.DDS_PROJECT) ? items.map((child) => {```